### PR TITLE
docs: Update documentation for installation

### PR DESCRIPTION
**Pull request by jecfish**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
 # Nightwatch Recorder Extension
 
 Nightwatch extension for Devtools that allows you to export tests directly from the recorder panel.
+
+# How to use it
```
Comment (jecfish) [inline Readme.md]:
```suggestion
# How to install and use it
```

File: Readme.md [hunk 2]
```diff
@@ -1,3 +1,13 @@
 # Nightwatch Recorder Extension
 
 Nightwatch extension for Devtools that allows you to export tests directly from the recorder panel.
```
Comment (jecfish) [inline Readme.md]:
```suggestion
Nightwatch extension for Devtools that allows you to export tests directly from the Chrome DevTools [Recorder panel](https://goo.gle/devtools-recorder).
```

File: Readme.md [hunk 3]
```diff
@@ -1,3 +1,13 @@
 # Nightwatch Recorder Extension
 
-Nightwatch extension for Devtools that allows you to export tests directly from the recorder panel.
+Nightwatch extension for Devtools that allows you to export tests directly from the Chrome DevTools [Recorder panel](https://goo.gle/devtools-recorder).
```
Comment (jecfish) [inline Readme.md]:
```suggestion
[Nightwatch Recorder]((https://chrome.google.com/webstore/detail/nightwatch-chrome-recorde/nhbccjfogdgkahamfohokdhcnemjafjk/related)) extension allows you to export tests directly from the Chrome DevTools [Recorder panel](https://goo.gle/devtools-recorder).
```

File: Readme.md [hunk 4]
```diff
@@ -1,3 +1,17 @@
 # Nightwatch Recorder Extension
 
-Nightwatch extension for Devtools that allows you to export tests directly from the recorder panel.
+[Nightwatch Recorder]((https://chrome.google.com/webstore/detail/nightwatch-chrome-recorde/nhbccjfogdgkahamfohokdhcnemjafjk/related)) extension allows you to export tests directly from the Chrome DevTools [Recorder panel](https://goo.gle/devtools-recorder).
+
+# How to install and use it
+
+1. Install the [extension](https://chrome.google.com/webstore/detail/nightwatch-chrome-recorde/nhbccjfogdgkahamfohokdhcnemjafjk/related) from Chrome Web Store.
```
Comment (vaibhavsingh97) [inline Readme.md]:
```suggestion
1. Install the [extension](https://chrome.google.com/webstore/detail/nightwatch-chrome-recorde/nhbccjfogdgkahamfohokdhcnemjafjk) from Chrome Web Store. 
```

File: Readme.md [hunk 5]
```diff
@@ -1,3 +1,17 @@
 # Nightwatch Recorder Extension
 
-Nightwatch extension for Devtools that allows you to export tests directly from the recorder panel.
+[Nightwatch Recorder]((https://chrome.google.com/webstore/detail/nightwatch-chrome-recorde/nhbccjfogdgkahamfohokdhcnemjafjk/related)) extension allows you to export tests directly from the Chrome DevTools [Recorder panel](https://goo.gle/devtools-recorder).
```
Comment (vaibhavsingh97) [inline Readme.md]:
```suggestion
[Nightwatch Recorder](https://chrome.google.com/webstore/detail/nightwatch-chrome-recorde/nhbccjfogdgkahamfohokdhcnemjafjk) extension allows you to export tests directly from the Chrome DevTools [Recorder panel](https://goo.gle/devtools-recorder).
```

File: Readme.md [hunk 6]
```diff
@@ -1,3 +1,17 @@
...
+2. Open a new tab in the browser.
+3. [Open DevTools](https://developer.chrome.com/docs/devtools/open/).
+4. Open an existing user flow or [record a new user flow](https://goo.gle/devtools-recorder) with the Recorder panel.
+5. [Export the user flow](https://developer.chrome.com/docs/devtools/recorder/reference/#export-flows) as Nightwatch test script. 
+  <img width="873" alt="Screen Shot 2022-08-17 at 9 31 15 AM" src="https://user-images.githubusercontent.com/5917927/185052858-44ba664a-33e4-48e3-83f3-83ff295d4d52.png">
+6. [Run the test script](https://nightwatchjs.org/guide/running-tests/using-the-cli-test-runner.html) with Nightwatch!
+
+# Export user flows as Nightwatch test scripts programmatically
+
+Use the [Nightwatch Chrome Recorder](https://github.com/nightwatchjs/nightwatch-chrome-recorder) CLI to Eeport user flows as Nightwatch test scripts programmatically.
```
Comment (vaibhavsingh97) [inline Readme.md]:
```suggestion
Use the [Nightwatch Chrome Recorder](https://github.com/nightwatchjs/nightwatch-chrome-recorder) CLI to Export user flows as Nightwatch test scripts programmatically. 
```

**Comment by vaibhavsingh97** [comment]
Thanks, @jecfish for updating the documentation 🙌

**Pull request merged**